packages/gestalt/src/SearchField.css

```diff
@@ -32,15 +26,38 @@
 }
 
 .input:focus {
-  background-color: #fff;
+  background-color: var(--white);
+  cursor: text;
```

Comment:
@nachofassini Do we need the `cursor: text`? Since this is a regular `<input />` field, I don't think we need it.

Reply:
Yes, we need it. Because in PDS they ask for `cursor: pointer` when hovering. Now i updated to set `cursor: text` when input is focused. That way we are not overwriting input cursor.